internal/reports/report_helpers.go

```diff
@@ -351,7 +328,7 @@ func buildTableReportBody(conv *internal.Conv, tableId string, issues map[string
 					}
 
 				case internal.AutoIncrementIndex:
-					str := fmt.Sprintf(" %s for Table %s and Column  %s", IssueDB[i].Brief, spSchema.Name, spColName)
+					str := fmt.Sprintf(" %s for Table '%s' and Column '%s'", IssueDB[i].Brief, spSchema.Name, spColName)
```

Comment:
wondering why the quotes have been added on all the logs. Is this for readability ? Has this been taken up while logging tables throughput the code base?

Reply:
This has been added for readability and to make it consistent across all the issues/warnings. It is not used for logging